Core/EventHandlers.lua

```diff
@@ -1637,6 +1637,48 @@ function R:OnEvent(event, ...)
 			end
 		end
 
+		--handles pets found in zskera vault chests
+		local vaultPets = {
+			"Bunbo",
+			"Berylmane",
+			"Brightfeather",
+		}
+		local vaultToys = {
+			"Obsidian Battle Horn",
+		}
+		local vaultChests = {
+			["Chain-Bound Strongbox"] = true,
+			["Crystal Basket"] = true,
+			["Hardened Chest"] = true,
+			["Mindless Slime"] = true,
+			["Searing Chest"] = true,
+			["Hardened Strongbox"] = true,
+			["Cart of Crushed Stone"] = true,
+			["Hardshell Chest"] = true,
+			["Titan Coffer"] = true,
+			["Void-Bound Strongbox"] = true,
+			["Inert Goo"] = true,
+			["Mysterious Chest"] = true,
+			["Supply Trunk"] = true,
+			["Shattered Crystals"] = true,
+			["Chest of Ice"] = true,
+			["Forgotten Lockbox"] = true,
+			["Slimy Goo"] = true,
+			["Wind-Bound Strongbox"] = true,
+			["Spun Webs"] = true,
+			["Obsidian Grand Cache"] = true,
+			["Frozen Coffer"] = true,
+		}
+		if Rarity.isFishing and Rarity.isOpening and Rarity.lastNode and vaultChests[Rarity.lastNode] then
```

Comment:
Should this not be using the localized version? (prefixed by `L`)

At least that's what all the other code is doing. For example:

```
-- Handle opening Timeless Chest
	if Rarity.isFishing and Rarity.isOpening and Rarity.lastNode and (Rarity.lastNode == L["Timeless Chest"]) then
	-- <the rest of the code>
```